flw/src/main/java/org/motechproject/nms/flw/domain/FrontLineWorker.java

```diff
@@ -0,0 +1,107 @@
+package org.motechproject.nms.flw.domain;
+
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Field;
+
+import org.motechproject.nms.language.domain.Language;
+import org.motechproject.nms.location.domain.District;
+
+import javax.jdo.annotations.Unique;
+
+/**
+ * Models data for simple records in a portable manner.
+ */
+@Entity
+public class FrontLineWorker {
+
+    @Field
+    private Long id;
```

Comment:
Eventually we'll have our id, MCTS id and one additional id.  I think the
FLW class will get flushed out when we do the csv import stories.

On Mon, Apr 20, 2015 at 12:57 PM, Lauren Lavoie <notifications@github.com>
wrote:

> In flw/src/main/java/org/motechproject/nms/flw/domain/FrontLineWorker.java
> <https://github.com/koshalt/mim/pull/46#discussion_r28722951>:
>
> > +import org.motechproject.mds.annotations.Entity;
> > +import org.motechproject.mds.annotations.Field;
> > +
> > +import org.motechproject.nms.language.domain.Language;
> > +import org.motechproject.nms.location.domain.District;
> > +
> > +import javax.jdo.annotations.Unique;
> > +
> > +/**
> > + * Models data for simple records in a portable manner.
> > + */
> > +@Entity
> > +public class FrontLineWorker {
> > +
> > +    @Field
> > +    private Long id;
>
> Who sets IDs? Do we not need them to be @Unique
> <https://github.com/Unique>?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/koshalt/mim/pull/46/files#r28722951>.
>